translations/messages+intl-icu.en_GB.yaml

```diff
@@ -102,9 +102,12 @@ home.no_tickets: 'No tickets'
 home.title: 'Welcome to Bileto'
 home.your_organizations: '{count, plural, one {Your organization} other {Your organizations}}'
 home.your_tickets: 'Your tickets'
-hours_formatter.hours: '{hours}h'
-hours_formatter.hours_and_minutes: "{hours}h\_{minutes}m"
-hours_formatter.minutes: '{minutes}m'
+hours_formatter.hours: '{hours} h'
+hours_formatter.hours_and_minutes: "{hours} h\_{minutes} m"
+hours_formatter.minutes: '{minutes} m'
+hours_formatter.hours.long: '{hours} hour'
+hours_formatter.hours_and_minutes.long: "{hours} hour\_{minutes} minute"
+hours_formatter.minutes.long: '{minutes} minute'
```

Comment:
What if there are more than 1 hour/minute? This will display "2 hour 30 minute" for instance. You must use the pluralization formatter from ICU (some information here: https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format). For instance:

```yaml
hours_formatter.hours.long: '{ hours, plural, one {1\_hour} other {#\_hours} }'
```

Note that I use the meta-character `\_` instead of a a simple space. `\_` is replaced by a non-breakable space. It means that if the string "1 hour" is placed at the end of a line, the unit will not be separated from the number. See the difference:

With a simple space:

!["1 heure 30 minutes" is split on two lines, 1 and "heure" are separated](https://github.com/Probesys/bileto/assets/107053378/738328b3-9f84-47f8-994a-39b98dba97d3)

With a non-breakable space:

!["1 heure 30 minutes" is put on a new line, 1 and "heure" are not separated](https://github.com/Probesys/bileto/assets/107053378/03cabfb5-e038-4e66-840f-f149e0dcfb5e)